tensorflow/python/training/initial_learning_rate_decay.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
@vrv do you have any suggestions on differentiating the names between the existing exponential function and the new exponential function?